app/pb_kits/playbook/pb_progress_pills/_progress_pills.html.erb

```diff
@@ -0,0 +1,19 @@
+<%= content_tag(:div,
+    id: object.id,
+    data: object.data,
+    class: object.classname) do %>
+    <div class="d-flex flex-column">
```

Comment:
We should remove these flexbox utility classes, we don't have them in Playbook. Keeping them may cause this kit to behave differently in Nitro.